src/index.ts

```diff
@@ -161,7 +161,10 @@ const tryRegExpsList = (input?: string): RegExp[] | undefined =>
 
 const { currentScript } = document;
 
-if (currentScript) {
+if (
+  currentScript &&
+  (currentScript.getAttribute('src')?.indexOf('sumologic') ?? -1 >= 0)
```

Comment:
@kkruk-sumo I think this should be:
```((currentScript.getAttribute('src')?.indexOf('sumologic') ?? -1) >= 0)```
If not, ` -1 >= 0` resolves first to `false`, and ends up working as: 
```(currentScript.getAttribute('src')?.indexOf('sumologic') ?? false)```

Reply:
@gsas2 Thank for reporting it. The fix should be in `1.4.3` :)